openvdb/points/AttributeSet.cc

```diff
@@ -101,7 +102,7 @@ AttributeSet::AttributeSet(const AttributeSet& attrSet, Index arrayLength)
         const Index stride = constantStride ? existingArray->stride() : existingArray->dataSize();
```

Comment:
Is it worth creating the lock here if it's null?

Reply:
Yes, that's a good idea.